pubspec.yaml

```diff
@@ -1,19 +1,19 @@
 name: graphs
-version: 2.2.0
+version: 2.2.1-dev
 description: Graph algorithms that operate on graphs in any representation
 repository: https://github.com/dart-lang/graphs
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: '>=2.18.0 <3.0.0'
 
 dependencies:
   collection: ^1.1.0
 
 dev_dependencies:
-  lints: ^1.0.0
+  lints: ^2.0.0
   test: ^1.16.0
 
   # For examples
-  analyzer: '>=2.0.0 <5.0.0'
+  analyzer: ^5.2.0
```

Comment:
Not for this PR, but it might be interesting to think of a simpler example that didn't require us to expose additional (dev) dependencies here.

Reply:
I agree. I also blame @natebosch 😛